### ISSUE TITLE
Create or change representation in XSLT for "Diple"

### ISSUE THREAD

**Issue by cgajjar05** [DCLP/dclpxsltbox#74]
Create or change representation in preview (XSLT) for "Diple".

**Comment by paregorios** [comment]
The LitPapText spreadsheet indicates that ```<milestone rend="diple" unit="undefined"/>``` "works in XML" but makes no statement as to XSLT effectiveness. I can find no examples of ```<milestone rend="diple" unit="undefined"/>``` in DCLP texts at present. @rla2118 @jcowey @HolgerEssler is this still a going concern, or should we close this ticket?

**Comment by rla2118** [comment]
@paregorios  I've put discussion of the XSLT representation of these "milestone rend" characters on the agenda for the next meeting.  I want to note though that we do not have Leiden+ for many of them. This seems to me more important than the XSLT concerns.

**Comment by paregorios** [comment]
@rla2118 yes, there are open tickets that Chirag made from the spreadsheet for the Leiden+ issues. I'm trying to at least get discussion going on resolution for all XSLT tickets this week so I can turn to resuscitating the Leiden+ tickets next week.

**Comment by rla2118** [comment]
This needs further discussion because the diple can be used for different purposes, e.g., for quotation marks, line filler, deletion, section breaks, etc.  Do we want to make semantic distinctions?

**Comment by paregorios** [comment]
@rla2118 ok I'll put this on the list for discussion at our next ticket review, which is on my calendar for 26 April (since we didn't get to this at the previous one). So, yes, there's the business of indicating the diple qua diple (as a mark or a character), then there's the question of encoding interpretation (an analogous case: line above indicating numerals).

Meantime, if @rogerbagnall @jcowey or @HolgerEssler have any thoughts about this, comments here would be welcome.

**Comment by rogerbagnall** [comment]
As with the macron, I do think we want to code things like this semantically. Isn't that what your presentation at ISAW on Friday would demand--tagging for meaning rather than arbitrary strings?

**Comment by rla2118** [comment]
I think so, too.

**Comment by paregorios** [comment]
Yes, of course we're tagging semantics. The point is not to conflate two related but distinct editorial decisions: the basic reading of a particular sign (which might be certain or less so) and the interpretation of that sign's function/significance in context (which might be separately contestable). Both are semantic concerns and both are worthy of clear expression in the encoding.

EpiDoc already has conventions for dealing with this sort of concern. Consider numerals, for example, as commonly in DDB:

```xml
<hi rend="supraline"><num value="2">β</num></hi>
```

In this particular case, we need to decide:

- Is it most appropriate to encode diple with ```<milestone>``` or with ```<g>```, given the TEI definitions for each?
- How do we want to mark up the interpretative semantics for its various uses?

**Comment by paregorios** [comment]
/me to research whether this is best encoded as milestone or g (note functions indicated by @rla2118 above, which should be expanded to include "uncertain" or "undefiined" or "not known" per @HolgerEssler ).

**Comment by paregorios** [comment]
Further discussion in #71 (still without resolution on the encoding issue). I'm presently of the opinion that we should use ```<g type="diple"/>``` and decorate or wrap it to indicate function as necessary/interesting.

**Comment by HolgerEssler** [comment]
I concur with @paregorios that this would be the best solution in xml - the issue of how best to implement it in Leiden+ should probably be deferred.

**Comment by paregorios** [comment]
Then ```<g type="diple"/>``` it is. Now we need to know what the papyrologists want to **see** in the HTML output. Right now one sees "((diple))", as (for example) in [TM 68614, column 2, line 7](http://litpap.info/dclp/68614).

Would you prefer to see something else @rogerbagnall @HolgerEssler @rla2118 @jcowey ?

**Comment by rogerbagnall** [comment]
I have no problem with this; it's searchable, and I don't think we'll ever come up with a single graphic representation of the diple. A documentary voice here, of course.
